### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_Inspect: Role for Some of the Controls(New Notebook,Revert to Checkpoint,Download as) is incorrect.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3998]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Navigating through Menubar.
4. Navigate to "File" menu and hit spacebar/enter to expand. 
5. Open Inspect Tool.
6. Verify Role for  Controls(New Notebook,Revert to Checkpoint,Download as).

**Actual:**
Role for Controls(New Notebook,Revert to Checkpoint,Download as) is given as Hyperlink/Link which is incorrect.
 * Narrator is also reading as "New Notebook"Link which is incorrect.
 * NVDA is also reading as "New Notebook"Link which is incorrect.

**Note:**
Same issue is observed for all Dropdowns present inside all Menus(File,Edit,View,Insert,Cell...etc).

**Expected:**
Role for Controls(New Notebook,Revert to Checkpoint,Download as) Should be Dropdown as they are Expanding.
 * Narrator should read as "New Notebook"Dropdown.
 * NVDA should read as "New Notebook"Dropdown.

**UserImpact:**
The user will not be able to know the significance of the control.

**Attachment:**
![a11y_azurenotebooks_jupyter_menubar_inspect](https://user-images.githubusercontent.com/39204909/45813822-4674cc80-bcf1-11e8-8d4f-5aeb3ae5ba61.png)

**Comment by takluyver** [comment]
Closed by #4690
